### DISCUSSION TITLE
Incompatible with Eurobank Gateway payment - doesnt show in Checkout

### DISCUSSION THREAD

**Discussion by seawind8** [woocommerce/woocommerce#45066] [WooCommerce, Blocks, and Site Editing]
## What do you like about the Cart & Checkout blocks?
Its easy to use and setup
## What do you think is missing from the Cart & Checkout blocks?
I will not use it since its not compatible with Eurobank Payment Gateway (official bank in Greece) the payment via credit card eurobank is enabled but in checkout page it says that there isnt any payment method. If you can fix that will be amazing.

**Comment by senadir** [comment]
Hi @seawind8 ,

The Cart and Checkout blocks are fairly new and there are a number of extensions and payment methods that are not integrated yet. Eurobank Payment Gateway is one of those extensions. 

We're not responsible for integrating third-party gateways, it's up to the provider to do so, and you as a customer can incentivize that.

You can follow [this document](https://docs.woocommerce.com/document/cart-checkout-blocks-support-status/) to keep up to date with the latest status of the blocks.

I'm going to close this issue as there's nothing actionable for it at the moment. We are aware of there being limited payment method support and it's on our roadmap to integrate WooCommerce owned extensions as well as encourage third party developers to integrate their own extensions.

Thanks for taking the time to create the issue and make sure we were aware!
